CHANGELOG.md

```diff
@@ -37,9 +37,14 @@
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
-## [2.0.5]
+## [2.1.0]
 
-> Release date: 2021/11/02
+> Release date: TBD
+
+**Note:** the admission webhook updates are _not_ applied automatically by the
```

Comment:
For easier application, maybe we could provide a copy-pasteable `kubectl patch --patch '{json}' validatingwebhookconfiguration kong-validations` instead of natural language instructions.